examples/run_all_notebooks.py

```diff
@@ -66,4 +67,6 @@
 
 
 print(df[['dataset','model','speed_up','score_diff']])
-df.to_csv("benchmark.csv")
+
+if args.execute:
+    df.to_csv("benchmark-v1.8.1.csv")
```

Comment:
can we automatically generate the filename based on the snapml version?
something like filename = "benchmark-v"+str(snapml.__version__)+".csv"?

Reply:
we could do that yes, will address that in a subsequent PR